apps/token/src/assets/env-config.js

```diff
@@ -1 +1,3 @@
 window._env_ = {};
+// eslint-disable-next-line no-undef, no-global-assign, no-native-reassign
+global = globalThis;
```

Comment:
chore (blocking): this solution won't work as this file is overrided when running in docker. Can we put this else where please?